backend/internal/service/server.go

```diff
@@ -65,6 +65,7 @@ func setupRoutes(app *fiber.App, config config.Config) {
 	app.Route("/reviews", func(r fiber.Router) {
 		reviewHandler := reviews.NewHandler(repository.Review, repository.User, repository.UserReviewVote)
 		r.Post("/", reviewHandler.CreateReview)
+		r.Get("/tag", reviewHandler.GetTags)
```

Comment:
super nit sorry but can we call the endpoint `/tags`

Reply:
fixed!